### DISCUSSION TITLE
Getting weird Error since 3 days

### DISCUSSION THREAD

**Discussion by TahBui** [crunchy-labs/crunchy-cli#361] [Q&A]
![image](https://github.com/crunchy-labs/crunchy-cli/assets/138880136/de160c78-7064-4dc8-a883-a70618f7c018)
This Error blocks all commands

**Comment by Serneum** [comment]
Lots of discussion around this has happened in https://github.com/crunchy-labs/crunchy-cli/issues/355

**Comment by bytedream** [comment]
This got fixed in v3.3.2, this or any more recent version do not have this anymore
